docs/content/doc/contributing/localization.de-de.md

```diff
@@ -1,17 +1,39 @@
 ---
-date: "2021-01-22T00:00:00+02:00"
-title: "Übersetzungs Richtlinien"
```

Comment:
Sure? The file name is localization.de-de.md

Reply:
Yup, and suddenly the German file is partly written in English?
I don't quite see how that is an improvement yet…